src/convert.ts

```diff
@@ -71,35 +85,46 @@ const blobToBase64 = (blob: Blob) =>
 const drawCanvas = ({ width, height }: { width: string, height: string }) => {
   const canvas = document.createElement("canvas")
   canvas.setAttribute("width", width)
-  canvas.setAttribute("height", `${parseFloat(height) + 45}px`)
+  canvas.setAttribute("height", px(Math.ceil(parseFloat(height))))
   return canvas
 }
 
+const createIframe = () => {
+  const iframe = document.body.appendChild(document.createElement("iframe"))
+  iframe.style.width = "100vw"
+  iframe.style.height = "100vw"
+  iframe.style.display = "none"
+  return iframe
+}
+
 const clickHandler = async () => {
   const rows = Array.from(document.querySelectorAll(".row"))
+  const iframe = createIframe()
   const images = rows.map(async (row) => {
     const table = row.querySelector("table")!
-    const svg = createSvg(table.cloneNode(true) as typeof table)
+
+    const svg = createSvg(table)
     svg.style.display = "none"
-    document.body.appendChild(svg)
+    iframe.contentDocument!.body.appendChild(svg)
 
     const data = new Blob([svg.outerHTML], {type: "image/svg+xml"})
     const base64Data = await blobToBase64(data) as string
 
     const tempImg = new Image()
+    tempImg.style.display = "none"
     tempImg.src = base64Data
 
-    const canvas = drawCanvas(window.getComputedStyle(table))
+    const canvas = drawCanvas(window.getComputedStyle(svg))
     const ctx = canvas.getContext("2d")!
     ctx.clearRect(0, 0, canvas.width, canvas.height)
-    await new Promise((resolve) => { tempImg.addEventListener("load", resolve) })
+    await new Promise(resolve => { tempImg.addEventListener("load", resolve) })
```

Comment:
The call to `addEventListener` should add `{ once: true }`, so that the event listener gets removed after it fires.